general/3_git2/Alena-Vorokh/HW-git-alena-vorokh.md

```diff
@@ -0,0 +1,77 @@
+#Homework Git Alena Vorokh
```

Comment:
In order for markdown to work properly you should put space after `#`. I recommend you change your headings through out your file(this point ab headings applies to all headings in .md file).